Modelica/Mechanics/MultiBody/Visualizers.mo

```diff
@@ -965,8 +966,9 @@ the last point of the parametrization coincide in this case.
   protected
     parameter SI.Radius rw = (width/2);
     parameter SI.Radius rCurvature2 = if rCurvature > rw then rCurvature else rw;
-    parameter SI.Radius h =     sqrt(1-(rw/rCurvature2)*(rw/rCurvature2))*rCurvature2;
-    parameter SI.Radius ri =    rTire-rCurvature2;
+    parameter Real kw = rw/rCurvature2 "Regularized width ratio (0...1)";
```

Comment:
```suggestion
    final parameter Real kw = rw/rCurvature2 "Regularized width ratio (0...1)";
```

Reply:
Fine to me.

Reply:
You can press the "Commit suggestion" button then.